wallet/i18n/vi.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
Please check Vietnamese Translations with Notional Team.

Reply:
@kjawadDeveloper2 Please coordinate with the Notional team to review Vietnamese Translation.

Reply:
@kjawadDeveloper2 can we close this issue if its approved and tested?